core/templates/dev/head/pages/story_editor/main_editor/StoryNodeEditorDirective.js

```diff
@@ -34,12 +34,12 @@ oppia.directive('storyNodeEditor', [
         '$scope', '$rootScope', '$uibModal', 'StoryEditorStateService',
         'StoryUpdateService', 'UndoRedoService', 'EVENT_STORY_INITIALIZED',
         'EVENT_STORY_REINITIALIZED', 'EVENT_VIEW_STORY_NODE_EDITOR',
-        'AlertsService',
+        'EXPLORATION_AND_SKILL_ID_PATTERN', 'AlertsService',
```

Comment:
This is not used anywhere, right? If so, can you remove this import?

Reply:
Done